### PR TITLE
[Library] Add dedicated web-component folder with examples and docs

### PR DESCRIPTION
**Pull request by juliawu**
Starting a web-component folder to hold external facing examples and documentation for the JS web components. Eventually, the web-component source code can be moved here as we move towards packaging it into a stand alone package.

The two README.md added form the start of dedicated documentation pages. We can reformat/refactor the documentation for clarity in followup PRs.

To preview the new folder structure, see: https://github.com/juliawu/datacommons-website/tree/67318322f95f220ce214ddf0bb89abaf2c30ae27/web-components

### CODE REVIEW

File: web-components/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    ></datacommons-line>
+  </body>
+</html>
+```
+For a full list of supported charts and attributes, see our
+[Web Component Documentation](./docs/README.md).
+
+### Finding DCIDs for Places and Variables
+
+Many web component attributes require [DCIDs](), as input. To find the DCID of a
```
Comment (dwnoble) [inline web-components/README.md:40]: Add link to DCIDs

File: web-components/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
+# Chart Component APIs
```
Comment (dwnoble) [inline web-components/docs/README.md:1]: Include "Data Commons" in title

File: web-components/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+```html
+<html>
+  <head>
+    <link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+    <link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+    <script src="https://datacommons.org/datacommons.js"></script>
+  </head>
+  <body>
+    <!-- Embedded charts will go here -->
```
Comment (dwnoble) [inline web-components/docs/README.md:25]: Can you add a real web component example here?

File: web-components/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+    <link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+    <script src="https://datacommons.org/datacommons.js"></script>
+  </head>
+  <body>
+    <!-- Embed a line chart -->
+    <datacommons-line
+      title="US Population Over Time"
+      place="country/USA"
+      variables='["Count_Person"]'
```
Comment (dwnoble) [inline web-components/README.md:30]: (not blocking for this pr): we should prioritize releasing the space-separated attribute converter

**Review by dwnoble — APPROVED** [review]
Looks great! Thanks Julia!

File: web-components/README.md (hunk 1 above)
Comment (juliawu) [reply web-components/README.md]: Good catch! Added.

File: web-components/docs/README.md (hunk 3 above)
Comment (juliawu) [reply web-components/docs/README.md]: Good idea. Added.

File: web-components/README.md (hunk 4 above)
Comment (juliawu) [reply web-components/README.md:30]: +1, once we make the change I'll update the examples here as well.

**Pull request merged**